src/clj/vanilla/db/core.clj

```diff
@@ -150,23 +150,113 @@
 
   (get-services vanilla-db)
 
+  (delete-all-services! vanilla-db)
 
+  (delete-service! vanilla-db {:id "1000"})
+  (delete-service! vanilla-db {:id "4000"})
+  (delete-service! vanilla-db {:id "7000"})
 
 
 
 
 
-  (delete-all-services! vanilla-db)
+  (drop-services-table vanilla-db)
 
-  (delete-service! vanilla-db {:id "1000"})
-  (delete-service! vanilla-db {:id "4000"})
-  (delete-service! vanilla-db {:id "7000"})
+  ())
 
 
+; ***Layout table rich comment***
+(comment
 
+  (hugsql/def-db-fns "sql/queries.sql")
+  (hugsql/def-sqlvec-fns "sql/queries.sql")
 
+  (create-layout-table-sqlvec vanilla-db)
 
-  (drop-services-table vanilla-db)
+  (create-layout-table vanilla-db)
```

Comment:
add a comment to make it clear that this line MUST be run at the repl to ensure the table(s) is/are built for the bootstrapped database.

Reply:
Done